src/containers/NFT/NFTHeader/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useContext, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import { useQuery } from 'react-query'
+import Loader from '../../shared/components/Loader'
+import '../../shared/css/nested-menu.css'
+import './styles.css'
+import SocketContext from '../../shared/SocketContext'
+import Tooltip from '../../shared/components/Tooltip'
+import { getNFTInfo } from '../../../rippled/lib/rippled'
+import { formatNFTInfo } from '../../../rippled/lib/utils'
+import {
+  analytics,
+  ANALYTIC_TYPES,
+  BAD_REQUEST,
+  HASH_REGEX,
+} from '../../shared/utils'
+import Details from './Details'
+import Settings from './Settings'
+
+const NFTHeader = (props) => {
```

Comment:
Don't use index.js for components. It makes it very hard to find components.

Reply:
huh, isnt it the convention throughout the repo to use this? or are we moving away from it?